api/v1alpha1/vaultdynamicsecret_types.go

```diff
@@ -20,6 +20,8 @@ type VaultDynamicSecretSpec struct {
 	Mount string `json:"mount"`
 	// Role in Vault to get the credentials for.
 	Role string `json:"role"`
+	// Revoke the credential when the resource is deleted.
```

Comment:
Should we also revoke credentials when they're rotated if Revoke==true?

Reply:
Oh that is a great question, let me look into that. I had assumed it'd just time out and that would be okay but we should probably also include that.

Reply:
Updated in 451a3268045f475de39ef228db34c225ff4b5512